components/ordered-list/README.md

```diff
@@ -7,8 +7,55 @@ OrderedList
 ```
 <!-- STORY -->
 
+### Usage
 
 
+Simple
+```js
+import ListItem from '@govuk-react/list-item';
+
+<OrderedList>
+  <ListItem>Lorem ipsum dolor sit.</ListItem>
+  <ListItem>Consectetur adipiscing elit.</ListItem>
+  <ListItem>Curabitur et libero nec.</ListItem>
+</OrderedList>
+```
+
+with Alpha
+```js
+import ListItem from '@govuk-react/list-item';
+
+<OrderedList listStyleType="lower-alpha">
```

Comment:
only need one example for this, as `listStyleType` is the CSS property `list-style-type`